bears/pug/PugLintBear.py

```diff
@@ -0,0 +1,233 @@
+                        prohibit_duplicate_attributes: bool=True,
+                        prohibit_html_text: bool=True,
+                        prohibit_id_attribute_with_static_value: bool=True,
+                        prohibit_id_literals_before_attributes: bool=True,
+                        prohibit_id_literals: bool=True,
+                        prohibit_legacy_mixin_call: bool=True,
+                        prohibit_multiple_line_breaks: bool=None,
+                        prohibit_spaces_inside_attribute_brackets: bool=True,
+                        prohibit_string_interpolation: bool=True,
+                        prohibit_tag_interpolation: bool=True,
+                        prohibit_specific_attributes: typed_list(str)=None,
+                        prohibit_specific_tags: typed_list(str)=None,
+                        enforce_class_literals_before_attributes: bool=False,
+                        enforce_class_literals_before_id_literals: bool=False,
+                        enforce_id_literals_before_attributes: bool=False,
+                        enforce_lower_case_attributes: bool=True,
+                        enforce_lower_case_tags: bool=True,
+                        require_spaces_inside_attribute_brackets: bool=False,
+                        require_strict_equality_operators: bool=True,
+                        validate_div_tags: bool=True,
+                        validate_extensions: bool=True,
+                        validate_self_closing_tags: bool=True,
+                        preferred_quotation: str="'",
+                        max_lines_per_file: int=None,
+                        puglint_config: str=''):
+        """
+        :param prohibit_block_expansion:
+            When ``True``, disallow any block expansion operators.
+            For example: If set to ``True``, this will throw a warning::
+
+                p: strong text
+                table: tr: td text
+
+        :param prohibit_class_attribute_with_static_value:
+            When ``True``, prefer class literals over class attributes with
+            static values.
+            For example: If set to ``True``, prefer ``span.foo`` over
+            ``span(class='foo')``.
+        :param prohibit_class_literals_before_attributes:
+            When ``True``, prefer all attribute blocks to be written before
+            any class literals.
+            For example: If set to ``True``, prefer
+            ``input(type='text').class`` over ``input.class(type='text')``.
+        :param prohibit_class_literals_before_id_literals:
+            When ``True``, prefer all ID literals to be written before any
+            class literals.
+            For example: If set to ``True``, prefer
+            ``input#id.class(type='text')`` over
+            ``input.class#id(type='text')``.
+        :param prohibit_class_literals:
+            When ``True``, disallow any class literals.
+            For example: If set to ``True``, prefer ``div(class='class')``
```

Comment:
i am kind of lost ``prohibit_class_attribute_with_static_value`` default value is True so you make use of class literals but here ``prohibit_class_literals`` you prohibit them.... isn't that contradicting?